server/single-server/src/main/resources/application.conf

```diff
@@ -34,7 +34,7 @@ logs {
   dataSourceClass = "slick.jdbc.DriverDataSource"
   connectionInitSql="set names utf8mb4"
   properties {
-    url = "jdbc:mysql:aurora://"${?SQL_LOGS_HOST}":"${?SQL_LOGS_PORT}"/"${?SQL_LOGS_DATABASE}"?autoReconnect=true&useSSL=false&serverTimeZone=UTC&socketTimeout=60000&useUnicode=true"
+    url = "jdbc:mysql://"${?SQL_LOGS_HOST}":"${?SQL_LOGS_PORT}"/"${?SQL_LOGS_DATABASE}"?autoReconnect=true&useSSL=false&serverTimeZone=UTC&socketTimeout=60000&useUnicode=true"
```

Comment:
Do we know what influence the `:aurora` part had?

Reply:
It influences the failover logic, leading to misleading error messages locally with the newest JDBC driver. Aurora has specifics for read replicas and failovers, and we don't need that locally.